developers/index.md

```diff
@@ -116,6 +115,6 @@ This script is specific for binding addons. Follow these steps to generate your
 
 1. Open Eclipse or your favorite editor and import your new binding project
 
-Now you can start developing your NEW binding. We're looking forward to your pull request!
+Now you can now start developing your NEW binding. We're looking forward to your pull request!
```

Comment:
Are both `now` intended here?
The first one looks weird after the change so far.

Reply:
Oops! I'll fix that in a minute.